### PR TITLE
feat: patch `btc` namespace price parameters

### PR DESCRIPTION
**Pull request by zone117x**
Closes https://github.com/blockstackpbc/devops/issues/540

This PR adds a patch to the `btc` namespace price parameters that get exported for Stacks 2.0. The goal is for names on the `btc` namespace to cost 2 STX per name.

@lgalabru can you verify the price parameters here are patched as intended? I ran into issues trying to test locally within the bns-test scripts.

### CODE REVIEW

**Review by jcnelson — COMMENTED** [review]
Can we confirm that the `.btc` namespace gets instantiated when the Stacks 2.0 node boots up?

**Comment by zone117x** [comment]
> Can we confirm that the `.btc` namespace gets instantiated when the Stacks 2.0 node boots up?

@jcnelson yes that is something we can either test manually or add to something like the automated migration test https://github.com/blockstack/stacks-blockchain/pull/2210

I'll make sure to do that before we get this merged 👍

**Comment by jcnelson** [comment]
Can you add it to #2210?  Then I'll approve it.

**Comment by zone117x** [comment]
@jcnelson are we okay to test this manually? It will take some time to add name testing to that PR, and we're trying to get a finalized v1 export image out.

**Comment by zone117x** [comment]
@lgalabru I was able to get the Stacks 2.0 Clarity BNS tests working here: https://github.com/blockstack/stacks-blockchain/commit/e23459f8ce94a5333cb49362af9224899e15c57a

And the parameters that result in 2 STX per name using the bns.clar contract are:
```js
{
  buckets: [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1],
  base: 1000,
  coeff: 200,
  noVowelDiscount: 1,
  nonAlphaDiscount: 1,
}
```

Does this look right to you?

**Comment by jcnelson** [comment]
Manual testing is fine, but you should be able to add an automated test to stacks-blockchain that just queries `.btc` once the genesis block is instantiated.  Would that be possible?

**Comment by zone117x** [comment]
> Manual testing is fine, but you should be able to add an automated test to stacks-blockchain that just queries `.btc` once the genesis block is instantiated. Would that be possible?

Yes, but we can't add Stacks 2.0 tests in this 1.0 branch. We first need to update master with a new `chainstate.txt` produced from the change in this Stacks 1.0 branch. Otherwise that step has to be done locally/manually.

**Comment by lgalabru** [comment]
Since we have https://github.com/blockstack/stacks-blockchain/pull/2286, what's our take on discarding this PR, and letting @MarvinJanssen update the btc price function, post migration?

**Comment by diwakergupta** [comment]
> Since we have #2286, what's our take on discarding this PR, and letting @MarvinJanssen update the btc price function, post migration?

I'd recommend keeping it in -- the PR is ready and there's no additional risk in having the namespace materialize with the correct price function in Stacks 2.0 genesis, but the converse is not true.

**Comment by zone117x** [comment]
This PR won't be ready until #2286 is finalized, afterwards we can finalize the price parameters in this PR.

**Review by jcnelson — APPROVED** [review]
As long as you're happy with the price parameters, this LGTM.

**Review by lgalabru — APPROVED** [review]
LGTM if this is the way!

**Review by kantai — APPROVED** [review]

**Pull request merged**
